h2o-core/src/main/java/water/api/schemas3/ModelMetricsRegressionGLMGenericV3.java

```diff
@@ -0,0 +1,18 @@
+package water.api.schemas3;
+
+import hex.ModelMetricsRegressionGLMGeneric;
+import water.api.API;
+
+public class ModelMetricsRegressionGLMGenericV3 extends ModelMetricsRegressionGLMV3<ModelMetricsRegressionGLMGeneric, ModelMetricsRegressionGLMGenericV3> {
+  @API(direction = API.Direction.OUTPUT, help="coefficients_table")
+  public TwoDimTableV3 _coefficients_table; // Originally not part of metrics, put here to avoid GenericOutput having multiple output classes.
```

Comment:
Ok, found one bug myself, this should have no underscore due to our conventions. I'll fix this (multiple classes):

- ModelMetricsRegressionGLMGenericV3
- ModelMetricsOrdinalGLMGenericV3
- ModelMetricsBinomialGLMGenericV3
- ModelMetricsMultinomialGLMGenericV3